tests/utils_tests/test_autoreload.py

```diff
@@ -360,11 +409,14 @@ def patch_autoreload(self, argv):
         return mock_call
 
     def test_manage_py(self):
+        script = Path('manage.py')
+        script.touch()
         argv = ['./manage.py', 'runserver']
         mock_call = self.patch_autoreload(argv)
         autoreload.restart_with_reloader()
         self.assertEqual(mock_call.call_count, 1)
         self.assertEqual(mock_call.call_args[0][0], [self.executable, '-Wall'] + argv)
+        script.unlink()
```

Comment:
`try: ... finally:` ?

Reply:
Or `self.addCleanup(script.unlink)` might be better?

Reply:
I went with `addCleanup()` — Thanks for mentioning that @orf.